### ISSUE TITLE
Why is the Cafe in the Library?

### ISSUE THREAD

**Issue by amenitykitten** [CHOMPStation2/CHOMPStation2#1261]
This is a very low priority issue, and mostly just something that bugs me personally, but...

It makes absolutely no sense for a Cafe to be attached to a library, considering in real life you're not supposed to bring beverages into a library because you could spill something on the things in the library and ruin them.
Also, it makes the Cafe a very 'out of the way' location, when a cafe should be more central (like it was when it was next to the bar).
And I understand that not everyone cares about the Cafe but I do care about the cafe, so it irritates me that it is in a very illogical and out of the way location that people wont really stumble across unless they specifically look for it, and also is attached to the one place that it makes no sense to be attached to.

It should be put back where it was, or moved to a more sensible location.
Possible areas:
- The place it was before, that is now the location of the Recreational Lounge.
- Attached to the Kitchen And/or Bar more directly.
- Above the Kitchen/And or Bar in the area labeled 'Cafeteria', or in the Botany Shop.
- Beside the Recreational Lounge, or attached to it.
- Near the dorms if people want to get a beverage without having to go all the way back to the bar.
All of the above make more sense than it being attached to the edge of the Library.

TLDR; 
The Problem:
![image](https://user-images.githubusercontent.com/78749284/107250944-0d447080-6a02-11eb-8643-641b561380e1.png)

The Proposed Solutions:
![image](https://user-images.githubusercontent.com/78749284/107251271-28af7b80-6a02-11eb-9dbf-3512c474a01b.png)
![image](https://user-images.githubusercontent.com/78749284/107251335-3d8c0f00-6a02-11eb-9fcc-1ce1a29fc437.png)

**Comment by Sharkmare** [comment]
there's not anything wrong with having a library with an attached coffee shop, that's not as far fetched as you think.

Sure in a massive big library that is a solid building dedicated only to books its gonna be rare to see.

But in the specific set up it is here where you have more something reminiscent of shopping centers with a shared middle seating area and stands to fetch food/drink whatever around the middle area.

putting it on dock 3 may be a bit much and too out of the way for anyone to notice that someone is actually using it which id think goes against your own intentions of making it more obvious again.

Personally, i think the simplest solution would be to just swap around the library office and the coffee area so that the shop is easily visible from the hallway just like the bar area. This would allow for easy visibility that someone is working in the shop or that people are gathered there,

**Comment by amenitykitten** [comment]
> there's not anything wrong with having a library with an attached coffee shop, that's not as far fetched as you think.

Sure, smaller CAFES dedicated to the niche of reading, do indeed have their own personal books they buy for their cafes, but large scale corporations would not have their cafe attached to their libraries due to the risk of wasting either paper resource, damaging books. This is a Corporate Space Station. The Library here is a public library, not a small cafe. If things get damaged, sure, they're easily replaced, but the costs over time if you keep spilling coffee on your books would see fit to be mitigated by "moving the cafe away from the library".

> Sure in a massive big library that is a solid building dedicated only to books its gonna be rare to see.

You are in space, on a massive space station. This is the space equivalent of a big massive library, theoretically, this library should hold every book for public missives, fiction or nonfiction, that the station should read. Some could be easily replaced, others would be costly. You have to look at this from a roleplay perspective of a space corporation that runs a research establishment and does not want to waste money by accidentally spilling coffee on books.

> But in the specific set up it is here where you have more something reminiscent of shopping centers with a shared middle seating area and stands to fetch food/drink whatever around the middle area.

The specific setup here is the setup of a small time bookstore or cafe with the niche that you can read while you drink. It does not fit the setup of 'we are on a research establishment, in space. We keep our things neat and tidy, and do not want to spill coffee on our books and research'. You will rarely see a shopping center that shares a food and drink area with a library for this same reason. And they typically have a good length of distance between them because they do not want their consumer items damaged.

> putting it on dock 3 may be a bit much and too out of the way for anyone to notice that someone is actually using it which id think goes against your own intentions of making it more obvious again.
> Personally, i think the simplest solution would be to just swap around the library office and the coffee area so that the shop is easily visible from the hallway just like the bar area. This would allow for easy visibility that someone is working in the shop or that people are gathered there,

You are thinking more of the 'simplest' solution, not the 'logical' solution. And that seems to be what is on your mind. The realistic solution is moreso that people frequent the central areas of the station more- and sure, putting it on the third _level_ of the station would make it less noticeable, but it would make more logical, sense. And, I actually do quite frequently see people go above the bar, and also quite frequently see people go to dorms, so putting it in that area would not be harmful in comparison to how many people visited it before, nor would it be intrusive.

**Comment by Shadow-Quill** [comment]
> Sure, smaller CAFES dedicated to the niche of reading, do indeed have their own personal books they buy for their cafes, but large scale corporations would not have their cafe attached to their libraries due to the risk of wasting either paper resource, damaging books. This is a Corporate Space Station. The Library here is a public library, not a small cafe. If things get damaged, sure, they're easily replaced, but the costs over time if you keep spilling coffee on your books would see fit to be mitigated by "moving the cafe away from the library".

Barnes & Noble, a bookseller with over 600+, usually has a Starbucks in their shops.
https://en.wikipedia.org/wiki/Barnes_%26_Noble#Food_service
 
> You are in space, on a massive space station. This is the space equivalent of a big massive library, theoretically, this library should hold every book for public missives, fiction or nonfiction, that the station should read. Some could be easily replaced, others would be costly. You have to look at this from a roleplay perspective of a space corporation that runs a research establishment and does not want to waste money by accidentally spilling coffee on books.

To be fair, we can literally print off an entire book (binding included) in less than a second.
 
> The specific setup here is the setup of a small time bookstore or cafe with the niche that you can read while you drink. It does not fit the setup of 'we are on a research establishment, in space. We keep our things neat and tidy, and do not want to spill coffee on our books and research'. You will rarely see a shopping center that shares a food and drink area with a library for this same reason. And they typically have a good length of distance between them because they do not want their consumer items damaged.

The fact we have a bar doesn't entirely fit with "we are on a research establishment, in space", where drunk crewmembers could ruin research.

> You are thinking more of the 'simplest' solution, not the 'logical' solution. And that seems to be what is on your mind. The realistic solution is moreso that people frequent the central areas of the station more- and sure, putting it on the third _level_ of the station would make it less noticeable, but it would make more logical, sense. And, I actually do quite frequently see people go above the bar, and also quite frequently see people go to dorms, so putting it in that area would not be harmful in comparison to how many people visited it before, nor would it be intrusive.

You forget that people who are heading to the dorms usually _don't stop_ going to the dorms. And people don't _tend_ to sit in an empty, out of the way place. They typically use the bar for sitting in.

**Comment by Sharkmare** [comment]
When talking about the horrid waste of paper.... one has to remember the fact that recycling of paper has to reach 1:1 ratios.

if you have a messy sheet of paper just feed it to a recycler to make a new clean sheet of paper ready to print.
And this technology is at a level where it's a thing you can just have on a laptop or tablet.

As for certain books being costly to replace... why?
Every single book should be digitally archived and printable on-demand, there should be no hard copies.

When it comes to "logical" placing I think you are relying too much on current-day tech when considering things.

**Comment by Darlantanis** [comment]
This is intentional. DM Kassc if you wish to discuss further.
